index.html

```diff
@@ -8,9 +8,24 @@
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
+    <!--Navigation section-->
     <section class="top">
+        <div class="mobile-nav">
+            <img src="./gallery/my logo.png" alt="" class="logo">
+            <i class="fa fa-bars hamurger"></i>
+        </div>
```

Comment:
- [ ] Please make sure that the hamburger button has a `nav` tag. The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices. :pray: